src/Frontend/Page/Overview.hs

```diff
@@ -175,6 +175,7 @@ instance ToHtml Tabs where
 -- * idea lists
 
 data ListItemIdeaContext  -- TODO: rename to 'ListLocation'?  'Context' is used for too many things.
+                          -- 'Position'?  i think my favorite so far would be `WhatList`.  ~~mf
```

Comment:
nah, i like the type, i just want to find a more awesome name.   oh, just noticed it's a TODO.

hum.

I'll rename it to `WhatListPage` and see if anybody will try to hurt me.